deploy/bav-spec.yaml

```diff
@@ -198,6 +198,103 @@ paths:
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws"
 
+  /token:
+    post:
+      operationId: postToken
+      summary: Exchange an Authorization Code for an Access Token
+      description: >-
+        Issue Access Token. OAuth2 standard endpoint which exchanges the authorization_code supplied and
+        returns a signed access_token which can be used directly in API calls as a Bearer token. Note
+        the service will only issue an access token once for any given session.
+      tags:
+        - Backend - Common CRI Lambda
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/IssueTokenRequest'
+      responses:
+        "200":
+          description: >-
+            Created - OAuth token is valid - Session details have been stored in the backend and a `session_id`
+            is returned.
+          headers:
+            Cache-Control:
+              schema:
+                type: "string"
+            Content-Type:
+              schema:
+                type: "string"
+            Strict-Transport-Security:
+              schema:
+                type: "string"
+            X-Content-Type-Options:
+              schema:
+                type: "string"
+            X-Frame-Options:
+              schema:
+                type: "string"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/IssueTokenResponse"
+        "401":
+          description: Unauthorized
+          headers:
+            Cache-Control:
+              schema:
+                type: "string"
+            Content-Type:
+              schema:
+                type: "string"
+            Strict-Transport-Security:
+              schema:
+                type: "string"
+            X-Content-Type-Options:
+              schema:
+                type: "string"
+            X-Frame-Options:
+              schema:
+                type: "string"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OAuthErrorResponse"
+        "500":
+          description: Internal Server Error
+          headers:
+            Cache-Control:
+              schema:
+                type: "string"
+            Content-Type:
+              schema:
+                type: "string"
+            Strict-Transport-Security:
+              schema:
+                type: "string"
+            X-Content-Type-Options:
+              schema:
+                type: "string"
+            X-Frame-Options:
+              schema:
+                type: "string"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OAuthErrorResponse"
+      x-amazon-apigateway-request-validator: "requestParamsOnly"
```

Comment:
We're only validating the params but we're only expecting a body to be sent?

Reply:
I have updated the value to both and re-deployed the stack and it looks good for now. Changed the value to both.

Reply:
Great thanks for checking!